src/lib/cdn.js

```diff
@@ -0,0 +1,201 @@
+/******************************************************************************
+Copyright (c) 2011, Google Inc.
+All rights reserved.
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
+    * Redistributions of source code must retain the above copyright notice,
+      this list of conditions and the following disclaimer.
+    * Redistributions in binary form must reproduce the above copyright notice,
```

Comment:
This line is a bit confusing to me. Not sure if an XPI counts as a "binary" and if we need to display this notice on Test Pilot site, or in the README, or some other special place.

Or, if we publish this separately on npm and include it as a dependency. Although possibly extreme overkill.